examples/guidelines-snap-tutorials.md

```diff
@@ -179,7 +254,7 @@ Syntax is regular [markdown syntax](https://guides.github.com/features/mastering
 
 ### Text
 
-For coherence in text inside code snippets, notes and survey with paragraphs, each end of line is reflected. A single end of line create a line return (*br*), while an end of line followed by an empty line will delimit pagraphs.
+For coherence in text inside code snippets, notes and survey with paragraphs, each end of line is reflected. A single end of line create a line return (*br*), while an end of line followed by an empty line will delimit paragraphs.
```

Comment:
To ensure the coherence of text inside code snippets, notes and a survey with paragraphs, how you end a line within these sections will affect the output. A single end of line will create a line return (*br*), while an end of line followed by an empty line will delimit paragraphs.